int32[] id
int32[] x
int32[] y
int32[] h
int32[] w 
--------------------------------
string[] name
